packages/templating-tools/package.js

```diff
@@ -16,7 +16,7 @@ Package.onUse(function(api) {
     // boilerplate-generator uses spacebars-compiler.)
     // XXX maybe uglify should be applied by this plugin instead of via magic
     // weak dependency.
-    'minifier-js@1.2.14'
+    'minifier-js@1.2.14 || 2.0.0'
```

Comment:
Ok, I tried removing it and commited to a [test branch](https://github.com/eagerestwolf/blaze/tree/test-minifier-js) and I am running a [Circle build](https://circleci.com/gh/eagerestwolf/blaze/40) now to see if everything passes or fails.